doc/libf3d/PLUGINS.md

```diff
@@ -1,6 +1,7 @@
 # Plugin SDK
 
-When calling `find_package(f3d REQUIRED COMPONENTS pluginsdk)` in CMake, a few macros are made available to you to generate a plugin which allow you to extend libf3d to support your own file format. Access to a f3d::vtkext VTK module is also provided if needed.
+Please follow instructions in the [F3D build guide](../dev/BUILD.md), make sure to build with `BUILD_SHARED_LIBS=ON` and to install the `plugin_sdk` component.
+You will then be able to call `find_package(f3d REQUIRED COMPONENTS pluginsdk)` in your plugin CMakeLists.txt, a few macros are made available to you to generate a plugin which allow you to extend libf3d to support your own file format. Access to a f3d::vtkext VTK module is also provided if needed.
```

Comment:
Speaking of `f3d::vtkext`, we'll have to generate the online documentation for it.

Reply:
https://github.com/f3d-app/f3d/pull/1321